protonvpn_cli/cli.py

```diff
@@ -435,14 +443,13 @@ def set_protonvpn_tier(write=False, init=False):
             # Check if the choice exists in the dictionary
             protonvpn_plans_selection[user_tier]
             if not init and user_tier == 5:
-                configure_cli()
+                return
             break
         except (KeyError, ValueError):
             print()
             print("[!] Invalid choice. Please enter the number of your plan.")
 
-    if write:
-
+    if not init:
```

Comment:
And last but not least, in all instances I've replaced the write boolean with the init boolean. :)